GIT/Test.md

```diff
@@ -0,0 +1,19 @@
+# Working on Conflicts
+
+Create a Repo
+
+Now create a simple js file(index.js) and some script
+
+window.alert("helloo, I'm Dharma");
+window.alert("hi, I'm Maniraj");
+
+Create a branch in the repo with Yourname/feature1
+
+Switch to this new branch,
+
+Edit the second message
+
+And try to merge this new branch with the main branch where you can see the conflicts.
```

Comment:
After merging this pr, you'll get conflicts in other pr because that pr was also trying to edit the same line